tutorials/vr/openvr/vr_starter_tutorial_part_two.rst

```diff
@@ -13,7 +13,7 @@ In this part of the VR starter tutorial series, we will be adding a number of sp
 This continues from where we left on in the last tutorial part, where we just finished getting the VR controllers working and defined a custom
```

Comment:
```suggestion
This continues from where we left off in the last tutorial part, where we finished getting the VR controllers working and defined a custom
```
Maybe I am missing context, but "leaving off" seems to fit better?
The word "just" is superfluous IMHO.